extensions/blocks/story/icon-customize.native.js

```diff
@@ -0,0 +1,10 @@
+/**
+ * WordPress dependencies
+ */
+import { Path, SVG } from '@wordpress/components';
+
+export default (
+	<SVG xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24">
+		<Path d="M2 6c0-1.505.78-3.08 2-4 0 .845.69 2 2 2 1.657 0 3 1.343 3 3 0 .386-.08.752-.212 1.09.74.594 1.476 1.19 2.19 1.81L8.9 11.98c-.62-.716-1.214-1.454-1.807-2.192C6.753 9.92 6.387 10 6 10c-2.21 0-4-1.79-4-4zm12.152 6.848l1.34-1.34c.607.304 1.283.492 2.008.492 2.485 0 4.5-2.015 4.5-4.5 0-.725-.188-1.4-.493-2.007L18 9l-2-2 3.507-3.507C18.9 3.188 18.225 3 17.5 3 15.015 3 13 5.015 13 7.5c0 .725.188 1.4.493 2.007L3 20l2 2 6.848-6.848c1.885 1.928 3.874 3.753 5.977 5.45l1.425 1.148 1.5-1.5-1.15-1.425c-1.695-2.103-3.52-4.092-5.448-5.977z" />
+	</SVG>
+);
```

Comment:
Re: this file here, I tried to export this:
```
   export { default as SvgIconCustomize } from './mobile/image/icon-customize';

```
in file [gutenberg/packages/components/src/index.native.js](https://github.com/WordPress/gutenberg/blob/master/packages/components/src/index.native.js)

but couldn't load it here in Jetpack, so, decided to copy the `SVGicon` definition file in the jetpack repo as well.

If there is a way we can do this without copying this file here, LMK 🙏